tests/components/test_input_boolean.py

```diff
@@ -22,20 +24,12 @@ def tearDown(self):  # pylint: disable=invalid-name
 
     def test_config(self):
         """Test config."""
-        self.assertFalse(input_boolean.setup(self.hass, {
-            'input_boolean': None
-        }))
-
-        self.assertFalse(input_boolean.setup(self.hass, {
-            'input_boolean': {
-            }
-        }))
-
-        self.assertFalse(input_boolean.setup(self.hass, {
-            'input_boolean': {
-                'name with space': None
-            }
-        }))
+        with self.assertRaises(vol.Invalid):
+            input_boolean.PLATFORM_SCHEMA(None)
```

Comment:
I prefer testing this using `bootstrap._setup_component` to follow the natural flow of discovering the validations and running them.